scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -161,26 +187,25 @@ def get_events_descriptions(self):
 
     def _read_single_event_rtt(self):
         id = int.from_bytes(
-            self._read_bytes(
-                self.config['rtt_data_channel'],
-                1),
+            self._read_bytes(1),
```

Comment:
I think you could speed it up even more by doing a single RTT operation `_read_bytes()` and then parsing the received data to get `id`, `buf` and other needed information.

Reply:
I meant underneath `_read_bytes()` there is `rtt_read()`. Instead of doing `rtt_read(1)` and then `rtt_read(4)` you could do `rtt_read(1+4)` and then read needed info from the received array. This is less overhead, important because we see that RTT can be a bottleneck.

Reply:
Discussed offline with @MarekPieta, agreed the rtt read size does not depend on argument of `_read_bytes()`